src/containers/Review/ReviewWrapper.tsx

```diff
@@ -105,7 +107,7 @@ const ReviewWrapper: React.FC<ReviewWrapperProps> = ({ structure }) => {
             orgName={org?.name as string}
           />
           <div id="review-home-content">
-            <ReviewProgress structure={structure} />
+            {!isMobile && <ReviewProgress structure={structure} />}
```

Comment:
Oh one more thing. The progress bar actually looks okay on mobile up to 3 items. Maybe we should keep it unless there are more that two stages (which makes 3 dots in the bar)?

![Screenshot 2023-11-07 at 12 28 33 PM](https://github.com/msupply-foundation/conforma-web-app/assets/5456533/2c787ae9-4a6e-4449-9c89-b39602eab1f3)

(But just reduce the top/bottom space around it)

Reply:
Actually, don't worry about it. It's still a bit cut off. More trouble than it's worth I think.

Reply:
Oh, I actually did this.  If more than 2 stages, will not show in mobile.